CONTRIBUTING.md

```diff
@@ -0,0 +1,128 @@
+# Overview
+
+This documents explains the processes and practices recommended for
+contributing enhancements to the Prometheus charm.
+
+- Generally, before developing enhancements to this charm, you should consider
+  [opening an issue ](https://github.com/canonical/prometheus-operator) explaining
+  your the use case.
```

Comment:
I would change: 

` explaining your the use case.` 

with:

` explaining your use case.`

Reply:
Thanks. Fixed.